xrpl4j-integration-tests/src/test/java/org/xrpl/xrpl4j/tests/environment/RippledContainer.java

```diff
@@ -84,7 +84,7 @@ public class RippledContainer {
    * No-args constructor.
    */
   public RippledContainer() {
-    try (GenericContainer<?> container = new GenericContainer<>("rippleci/rippled:2.2.0")) {
+    try (GenericContainer<?> container = new GenericContainer<>("rippleci/rippled:2.3.0")) {
```

Comment:
```suggestion
    try (GenericContainer<?> container = new GenericContainer<>("rippleci/rippled:latest")) {
```

Reply:
`latest` is `2.3.1` which has an issue that prevents it from starting up on mac silicon unfortunately

Reply:
`latest` is now v2.4; verified it works on Mac silicon.